glommio/src/sys/mod.rs

```diff
@@ -558,3 +573,163 @@ pub(crate) fn shutdown(raw: RawFd, how: Shutdown) -> io::Result<()> {
         _ => Ok(()),
     }
 }
+
+/// Stores information about IO performed in a specific ring
+#[derive(Default, Debug, Copy, Clone)]
+pub struct RingIoStats {
```

Comment:
`sys` is not a public module, so those structs that are maent to be public need to be reexported somewhere.

Note that in the docs, where you link this, you are referring to `crate::sys::IoStats`, which I believe is broken